### PR TITLE
Added solution to CSES problem coin piles and edited the readme

### PR DESCRIPTION
**Pull request by Suyyashaa**

### CODE REVIEW

**Review by aksh1407 — CHANGES_REQUESTED** [review]
Kindly mention the programming language of your solution in the readme file correctly.
You submitted the solution in C++ but mentioned Python.

**Comment by Suyyashaa** [comment]
Hey! can you check if the error is fixed now?

File: Competitive Programming/Solutions/CSES/Introductory Problems/readme.md [hunk 1]
```diff
@@ -14,3 +14,8 @@
 ---
 
 *[Solution (Python)](https://github.com/1gne0u5/Learning-Resources/blob/master/Competitive%20Programming/Solutions/CSES/Introductory%20Problems/weirdAlgorithm.py)*
+
+**Coin Piles**
+---
+
+*[Solution (Python)](https://github.com/1gne0u5/Learning-Resources/blob/master/Competitive%20Programming/Solutions/CSES/Introductory%20Problems/coinPiles.cpp)*
```
Comment (aksh1407) [inline Competitive Programming/Solutions/CSES/Introductory Problems/readme.md:21]:
```suggestion
*[Solution (C++)](https://github.com/1gne0u5/Learning-Resources/blob/master/Competitive%20Programming/Solutions/CSES/Introductory%20Problems/coinPiles.cpp)*
```
Change the language mentioned to **C++** and resolve the merge conflicts.

**Review by aksh1407 — CHANGES_REQUESTED** [review]

**Review by aksh1407 — APPROVED** [review]
Great! 👍🏼

**Pull request merged**